deploy-manage/toc.yml

```diff
@@ -456,16 +456,21 @@ toc:
           - file: security/secure-your-eck-installation.md
       - file: security/secure-your-cluster-deployment.md
         children:
+          - file: security/security-certificates-keys.md
+          - file: security/set-up-minimal-security.md
           - file: security/secure-cluster-communications.md
             children:
-              - file: security/secure-http-communications.md
-              - file: security/k8s-transport-settings.md
-              - file: security/security-certificates-keys.md
+              - file: security/set-up-basic-security.md
+              - file: security/set-up-basic-security-plus-https.md
+              - file: security/updating-certificates.md
                 children:
-                  - file: security/updating-certificates.md
-                    children:
-                      - file: security/same-ca.md
-                      - file: security/different-ca.md
+                  - file: security/same-ca.md
+                  - file: security/different-ca.md
+              - file: security/kibana-es-mutual-tls.md
```

Comment:
same grouping/applicability note here - unclear that this is just self-managed from the sidebar

Reply:
I totally agree with that, if you can provide some navigation_titles for your suggestion (here and in the other item) that would be awesome :)

Reply:
done, let's see how it looks